EventFiltering/Zorro.h

```diff
@@ -44,6 +45,11 @@ class Zorro
   bool isNotSelectedByAny(uint64_t bcGlobalId, uint64_t tolerance = 100);
 
   void populateHistRegistry(o2::framework::HistogramRegistry& histRegistry, int runNumber, std::string folderName = "Zorro");
+  TH2* mZorroHisto = nullptr;
+  TH2* mToiHisto = nullptr;
+  void populateHist(int runNumber);
```

Comment:
I would change this function to be called `populateExternalHists` and pass to the function the pointers to the histograms to be populated